src/PrestaShopBundle/DependencyInjection/Compiler/CommandAndQueryCollectorPass.php

```diff
@@ -67,4 +67,16 @@ private function findCommandsAndQueries(ContainerBuilder $container)
 
         return $commands;
     }
+
+    /**
+     * update messenger tags allowing the recognition of handlers by symfony
+     */
+    private function updateMessengerTags(ContainerBuilder $container, array $handlers): void
```

Comment:
We should add the tag in every service definition instead of creating this. That avoids complexity and maintenance.

Reply:
I do not think so. If tomorrow symfony changes its tags, we will have to change the tags in one place. Also if we want to use symfony messenger for another use, there will be misunderstanding. Also it makes us gain in readability, we use 2 lines instead of 3, because the method is always the same